Leanplum-SDK/Classes/LPInbox.h

```diff
@@ -28,6 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - LPInboxMessage interface
 
+NS_SWIFT_NAME(InboxMessage)
```

Comment:
If the Inbox is LeanplumInbox, I dont like InboxMessage not having a prefix.

Reply:
I'm thinking to rename it to LeanplumInbox and LeanplumInbox.Message ?